wgsl/index.bs

```diff
@@ -4243,6 +4243,9 @@ the test name.
 * v-0026: The case selector values must have the same type as the selector expression.
 * v-0027: A literal value must not appear more than once in the case selectors for a switch statement.
 * v-0028: A fallthrough statement must not appear as the last statement in last clause of a switch.
+* v-0030: If a runtime-sized array appears inside the store type for a variable, then that variable
+          store type must be for a storage buffer variable.
+* v-0031: A runtime-sized array must not be the store type of a variable.
```

Comment:
wouldn't this contradict V-0015? If there is a variable with a store type = runtime-sized array, then surely it's no longer "only appear as the last member of a struct"

Reply:
There's a difference between being the store type for a variable and being contained in the store type of a variable.

The "store type" is the type after the colon in the variable declaration.  Nothing else.

This rule bans:
```
type RTArr = [[stride(4)]] array<f32>;
[[set(0),binding(1)]] var<storage> blah : RTArr; 
```

Reply:
In your code, the runtime-sized array appears not "as the last member of a struct", so it contradicts V-0015 already. Am I reading it wrong, still?

Reply:
Let's consider this example:

[[block]]
struct Foo {
  [[offset (0)]] a : [[stride (4)]] array<f32>;
};
[[set(0), binding(1)]] var<storage> buf: Foo;

Reply:
Could you elaborate, please?

Reply:
@dneto0 says my example is valid. sorry for the confusion.

Reply:
Right.  I think I see the redundancy between v-0015 and v-0031.